### PR TITLE
PLAT-2264 - Fix jumping behavior the table

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-2264)

### CODE REVIEW

**Review by mihaelpalfi — APPROVED** [review]
Looks good. Remember that if you specifically dont set the property type it will not translate the properties correctly to react. But for this $ properties its not needed right!
NIT: Still see some jumping when switching tabs. But this is probably another issue.

File: src/components/zen-table-cell/zen-table-cell.tsx [hunk 1]
```diff
@@ -63,6 +63,27 @@ export class ZenTableCell {
     this.parentRow().selected = (event.target as HTMLZenCheckboxElement).checked;
   }
 
+  transscriptParentRowProps(): void {
```
Comment (homer0) [inline src/components/zen-table-cell/zen-table-cell.tsx:66]: Being reaaaally picky here, sorry: transcript has only one `s`, and in any case, what do you think about `inherit` or `load` :P?

**Review by homer0 — APPROVED** [review]
@mihaelpalfi Don't worry about the props, the issue is when they have the `readonly`, but he removed it.

@chavlji awesome work 🤘 - just one small comment about naming :P (sorry!)

File: src/components/zen-table-cell/zen-table-cell.tsx (hunk 1 above)
Comment (chavlji) [reply src/components/zen-table-cell/zen-table-cell.tsx]: :) Np, fixed.

**Pull request merged**